### PR TITLE
Update header topbar with new design

### PR DESCRIPTION
**Pull request by azangru**
## Type
- Improvement to existing feature

## Related JIRA Issue(s)
- https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-803
- https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-804
- https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-805

## Description
- Update app topbar with the new logotype
- Remove the "e!" Ensembl logo
- Remove the video part of the HelpPopupButton
- Updated Home icon used in the topbar (the house image moved within the svg canvas)
- HeaderButtons component removed, because it's no longer used
- Update the icon for button linking to species page in HomepageAppLinks component

_Note:_ final style updates of the topbar include the latest Andrea's design changes (no buttons in the top right corner; home button moved to the top left corner)

## Deployment URL
http://update-topbar.review.ensembl.org/

## Views affected
All

### CODE REVIEW

File: src/ensembl/src/shared/components/error-screen/GeneralErrorScreen.tsx [hunk 1]
```diff
@@ -16,7 +16,10 @@
 
 import React from 'react';
 
-import { HomeLink, ReleaseVersion, Copyright } from 'src/header/Header';
+import { ReleaseVersion, Copyright } from 'src/header/Header';
+// import { HomeLink } from 'src/header/header-buttons/HeaderButtons';
```
Comment (msmanoj) [inline src/ensembl/src/shared/components/error-screen/GeneralErrorScreen.tsx:20]: Can this be removed? also on live 31

File: src/ensembl/src/shared/components/error-screen/GeneralErrorScreen.tsx [hunk 2]
```diff
@@ -25,10 +28,12 @@ import generalErrorImage1 from './images/general-error-1.jpg';
 import generalErrorImage2 from './images/general-error-2.jpg';
 import generalErrorImage3 from './images/general-error-3.jpg';
```
Comment (msmanoj) [inline src/ensembl/src/shared/components/error-screen/GeneralErrorScreen.tsx:29]: Can these be moved before the styles import?

File: src/ensembl/src/shared/components/help-popup/HelpPopupButton.tsx [hunk 3]
```diff
@@ -24,7 +23,6 @@ import HelpPopupBody from './HelpPopupBody';
 import { HelpAndDocumentation } from 'src/shared/components/app-bar/AppBar';
 
 import { ReactComponent as HelpIcon } from 'static/img/launchbar/help.svg';
-import { ReactComponent as VideoIcon } from 'static/img/shared/video.svg';
```
Comment (msmanoj) [inline src/ensembl/src/shared/components/help-popup/HelpPopupButton.tsx:27]: Should we delete this svg if it is no longer used anywhere?

File: src/ensembl/src/shared/components/error-screen/GeneralErrorScreen.tsx (hunk 1 above)
Comment (azangru) [reply src/ensembl/src/shared/components/error-screen/GeneralErrorScreen.tsx]: I've removed it. In fact, I've now removed the whole HeaderButtons component.

File: src/ensembl/src/shared/components/help-popup/HelpPopupButton.tsx (hunk 3 above)
Comment (azangru) [reply src/ensembl/src/shared/components/help-popup/HelpPopupButton.tsx:27]: It is used in the popup itself:

image

**Comment by azangru** [comment]
There is an `<Account />` component in the Header that's now unused. I suggest we remove it, and the redux code associated with it. Don't know whether in this PR or in a separate one.

File: src/ensembl/src/shared/components/help-popup/HelpPopupButton.tsx (hunk 3 above)
Comment (msmanoj) [reply src/ensembl/src/shared/components/help-popup/HelpPopupButton.tsx:27]: Ah yes! Sorry.

File: src/ensembl/src/shared/components/error-screen/GeneralErrorScreen.tsx (hunk 2 above)
Comment (azangru) [reply src/ensembl/src/shared/components/error-screen/GeneralErrorScreen.tsx:23]: ✔️

File: src/ensembl/src/header/Header.scss [hunk 4]
```diff
@@ -6,39 +6,72 @@
...
+  &Left {
+    display: flex;
+    align-items: center;
+    line-height: 1;
+  }
+
+  &LeftTextBlock {
+    display: flex;
+    align-items: center;
+    margin-top: 1px;
```
Comment (msmanoj) [inline src/ensembl/src/header/Header.scss:22]: This definition here only seems to affect the logotype so can we can remove this and increase the `margin-top` value on line 38 to `3px`?

Comment (azangru) [reply src/ensembl/src/header/Header.scss]: Yeah, the way the combined effect of the several flexboxes the topbar is very strange now. I've moved the margin-top to the `logotypeWrapper` class.

**Comment by azangru** [comment]
Updated a button on the Home page — it's now an "i" for "info" rather than a house; and it also links to its related species:

**Comment by msmanoj** [comment]
> There is an `<Account />` component in the Header that's now unused. I suggest we remove it, and the redux code associated with it. Don't know whether in this PR or in a separate one.

I think it is better to leave it to a separate PR at this point.

**Review by msmanoj — APPROVED** [review]

**Review by imransl — APPROVED** [review]

**Pull request merged**